go-controller/pkg/metrics/master.go

```diff
@@ -246,6 +250,32 @@ var metricPortBindingUpLatency = prometheus.NewHistogram(prometheus.HistogramOpt
 	Buckets:   prometheus.ExponentialBuckets(.01, 2, 15),
 })
 
+var metricNetworkProgramming prometheus.ObserverVec = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "network_programming_duration_seconds",
+	Help:      "The duration to apply network control plane configuration",
```

Comment:
Help: "The duration to apply network configuration for a kind (e.g. pod, service, networkpolicy). Configuration includes add, update and delete events combined in a single histogram."
^ the best I can come up with